airflow/analytical_platform/standard_operator.py

```diff
@@ -39,7 +39,10 @@ def __init__(
 
         for k, v in std_envs.items():
             if k not in env_vars:
-                env_vars[k] = v
+                env_vars[k] = str(v)
```

Comment:
Calling `str(v)` here seems unnecessary since you already define the values as strings, and then loop over `env_vars` below to convert any non-string values to strings. However, doesnt really hurt to do it either.